Assets/Mirror/Runtime/NetworkClient.cs

```diff
@@ -275,30 +246,12 @@ internal void Update()
 
         internal void RegisterHostHandlers(INetworkConnection connection)
         {
-            connection.RegisterHandler<ObjectDestroyMessage>(OnHostClientObjectDestroy);
-            connection.RegisterHandler<ObjectHideMessage>(OnHostClientObjectHide);
-            connection.RegisterHandler<NetworkPongMessage>(msg => { });
-            connection.RegisterHandler<SpawnMessage>(OnHostClientSpawn);
-            // host mode reuses objects in the server
-            // so we don't need to spawn them
-            connection.RegisterHandler<ObjectSpawnStartedMessage>(msg => { });
-            connection.RegisterHandler<ObjectSpawnFinishedMessage>(msg => { });
-            connection.RegisterHandler<UpdateVarsMessage>(msg => { });
-            connection.RegisterHandler<RpcMessage>(OnRpcMessage);
-            connection.RegisterHandler<SyncEventMessage>(OnSyncEventMessage);
+            //Currently no host related messages to register
```

Comment:
So why keeping this method?

Reply:
I didn't want to remove it and make its placement ambiguous in cases of future use.